bin/setup

```diff
@@ -11,7 +11,9 @@ init (){
   verify_ruby_version && verify_node_version
   verify_bundled_gems && verify_yarn_packages
   verify_foreman
-  verify_postgres && verify_database_migrations
+  verify_postgres && verify_database_migrations &&
+  cleanup_database
```

Comment:
Not sure if this is the right thing to do here, but the script before this change was causing an error that looked like:
```
psql:/Users/atharva/simple-server/db/structure.sql:48: ERROR:  type "gender_enum" already exists
rake aborted!
failed to execute:
psql --set ON_ERROR_STOP=1 --quiet --no-psqlrc --file /Users/atharva/simple-server/db/structure.sql simple-server_test6

Please check the output above for any errors and make sure that `psql` is installed in your PATH and has proper permissions.
```

This was because the `structure.sql` file was being run twice. I assume a cleanup was the intended step here, before running the parallel tasks.